### ISSUE TITLE
Fail to install Xcode

### ISSUE THREAD

**Issue by uhooi** [RobotsAndPencils/XcodesApp#137]
**Describe the bug**
I can't install Xcode 12.5.  
The same goes for Xcode 12.4, 12.2, and others.

```
An error occured

The archive "Xcode-12.5.0+12E262.xip" is damaged and can't be expanded.
```

**To Reproduce**
Click the [INSTALL] button.

**Expected behavior**
Xcode will be installed.

**Screenshots**
<img width="1012" alt="スクリーンショット 2021-06-07 15 50 48" src="https://user-images.githubusercontent.com/21194714/120972284-8d0ffb00-c7a8-11eb-9248-b15abd5f77cf.png">

Advanced  
<img width="552" alt="スクリーンショット 2021-06-07 16 14 14" src="https://user-images.githubusercontent.com/21194714/120974787-820a9a00-c7ab-11eb-88dc-5f6eca253be3.png">

**Version**
- MacBook Air (M1, 2020)
 - OS: macOS Big Sur 11.2.3
 - Xcodes: 1.0.5 (6)

<img width="698" alt="スクリーンショット_2021-06-07_15_55_20" src="https://user-images.githubusercontent.com/21194714/120972856-28a16b80-c7a9-11eb-9fc4-b89b9a410f2a.png">

<img width="580" alt="スクリーンショット 2021-06-07 15 54 42" src="https://user-images.githubusercontent.com/21194714/120972426-b2046e00-c7a8-11eb-93e5-e79f027d15dc.png">

**Comment by uhooi** [comment]
Installing the helper and restarting the Mac did not fix the problem.  
<img width="552" alt="スクリーンショット 2021-06-07 16 20 18" src="https://user-images.githubusercontent.com/21194714/120975590-605de280-c7ac-11eb-9d29-9346b4a5298e.png">

**Comment by uhooi** [comment]
Downloading command line developer tools did not fix the problem.

```
xcode-select: note: no developer tools were found at '/Applications/Xcode.app', requesting install. Choose an option in the dialog to download the command line developer tools.
```

<img width="498" alt="スクリーンショット 2021-06-07 21 59 00" src="https://user-images.githubusercontent.com/21194714/121030074-1d1f6600-c7e4-11eb-85cc-52373fcd6adb.png">
<img width="498" alt="スクリーンショット 2021-06-07 22 31 59" src="https://user-images.githubusercontent.com/21194714/121030076-1f81c000-c7e4-11eb-98f8-fa9a89c72246.png">

**Comment by uhooi** [comment]
Bumping macOS from 11.2.3 to 11.4  did not fix the problem.
<img width="668" alt="スクリーンショット 2021-06-08 9 35 37" src="https://user-images.githubusercontent.com/21194714/121104534-37d1f900-c83d-11eb-803a-83ea6fe787ca.png">

**Comment by MattKiazyk** [comment]
Hi @uhooi 

I just released v1.1 that has better support for M1

check for updates in the menu to install the new version

Thanks

**Comment by uhooi** [comment]
Sorry, I had signed in with an Apple ID that was not registered with the Apple Developer Program.  
<img width="440" alt="スクリーンショット_2021-06-08_9_41_24" src="https://user-images.githubusercontent.com/21194714/121105083-5258a200-c83e-11eb-8319-4c6f046d6237.png">

I can install Xcode.  
<img width="900" alt="スクリーンショット 2021-06-08 9 46 11" src="https://user-images.githubusercontent.com/21194714/121105103-62708180-c83e-11eb-9d78-1b5fb60a5725.png">

I close this issue.
